resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -0,0 +1,415 @@
+---
+
+- slug: trimmomatic-single
+  name: trimmomatic (single-end)
+  data_name: '{{ reads.fastq.0.file|default:"?" }}'
+  version: 0.0.1
+  type: data:reads:fastq:single:trimmomatic
+  category: analyses
+  flow_collection: sample
```

Comment:
Yes,, this it required if you want to include Data object into Sample. It will be added to existing Sample if one exist and new one will be created otherwise.